src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceData/CounterSetInstanceCounterDataSet.cs

```diff
@@ -162,16 +162,16 @@ internal CounterSetInstanceCounterDataSet(CounterSetInstance thisInst)
 
         public void Dispose()
         {
-            Dispose(true);
+            DisposeThis();
             GC.SuppressFinalize(this);
         }
 
         ~CounterSetInstanceCounterDataSet()
         {
-            Dispose(false);
+            DisposeThis();
         }
 
-        private void Dispose(bool disposing)
+        private void DisposeThis()
```

Comment:
Nit: DisposeCore instead of DisposeThis?

Reply:
changed